StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift

```diff
@@ -74,8 +83,8 @@ class ConnectWebView: WKWebView {
 @available(iOS 15, *)
 private extension ConnectWebView {
     // Opens the given navigation in a PopupWebViewController
-    func openInPopup(configuration: WKWebViewConfiguration,
-                     navigationAction: WKNavigationAction) -> WKWebView? {
+    func openInPopupWebViewController(configuration: WKWebViewConfiguration,
+                                      navigationAction: WKNavigationAction) -> WKWebView? {
```

Comment:
Renamed to distinguish between opening a SafariVC popup vs. `PopupWebViewController`